src/components/Authenticated/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { SvgXml } from 'react-native-svg';
+import { Button } from '..';
```

Comment:
```suggestion
import Button from '../Button';
```
Evitar Require cycle: src/components/index.ts -> src/components/Authenticated/index.tsx -> src/components/index.ts